webapp/channels/src/components/drafts/drafts.tsx

```diff
@@ -46,7 +70,22 @@ function Drafts({
         return () => {
             dispatch(unsuppressRHS);
         };
-    }, []);
+    }, [dispatch]);
+
+    const handleSwitchTabs = useCallback((key) => {
+        switch (key) {
+        case 0:
+            if (!isDraftsTab) {
+                history.push(`/${match.params.team}/drafts`);
+            }
```

Comment:
There are only two states, so we can simplify things by using just one variable (`isScheduledPostsTab`). Similar with the tabs, instead of a swtich, we can just use an if (unless there is any case where a different key could be sent).

Reply:
Done.